data/mode.txt

```diff
@@ -1 +1 @@
-extended
```

Comment:
Nope, that's temporary. I have the server gamemode set to Shadowling for testing.